website/src/components/toc.tsx

```diff
@@ -60,6 +61,31 @@ const CollectionTOC = () => {
           </Fragment>
         ) : null
       )}
+      <div className={css.noticeText}>
+      <CreativeCommonsBy
+        title="Cherokees Writing the Keetoowah Way"
```

Comment:
If we can have different authors depending on the collection, we could probably add author information into our database